reactor-netty-core/src/main/java/reactor/netty/tcp/SslProvider.java

```diff
@@ -224,6 +224,17 @@ public interface Builder {
 
 	public interface SslContextSpec {
 
+		/**
+		 * SslContext builder that provides, specific for the protocol, default configuration
+		 * e.g. {@link DefaultSslContextSpec}, {@link TcpSslContextSpec} etc.
+		 * The default configuration is applied before any other custom configuration.
+		 *
+		 * @param spec SslContext builder that provides, specific for the protocol, default configuration
+		 * @return {@literal this}
+		 * @since 1.2.0
+		 */
+		Builder sslContext(GenericSslContextSpec<?> spec);
+
```

Comment:
I just wonder why we keep the old signature for `sslContext` method that takes as argument a ProtocolSslContextSpec ?

```
		Builder sslContext(ProtocolSslContextSpec spec);
```

The `ProtocolSslContextSpec` class is now extending `GenericSslContextSpec<SslContextBuilder>`, so passing a ProtocolSslContextSpec to the new signature would work.

I may miss something obvious here, but why the old `Builder sslContext(ProtocolSslContextSpec spec)` signature is not removed ? maybe it's just because the old signature was not deprecated so far ?

Reply:
We can deprecated it only :(